src/js/config/Config.js

```diff
@@ -5,7 +5,7 @@ import ConfigTest from "./Config.test.js";
 var Config = {
   analyticsKey: "51ybGTeFEFU1xo6u10XMDrr6kATFyRyh",
   acsAPIPrefix: "/acs/api/v1",
-  applicationRenderDelay: 1000,
+  applicationRenderDelay: 2000,
```

Comment:
☝️  Loading the UI already takes quite some time as many of the assets and JavaScript resources are rather large, which is bad for the user experience as studies conducted by Google, Amazon, Facebook and the likes have shown. Adding any delay to the already long loading time is counterproductive and we should try remove the need for any loading screen instead of investing time to implement a nice screen and further  increasing the delay.

Reply:
I agree that we should aim for the fastest possible experience.  With that said, if we are going to display a preloader prior to rendering the login screen, having the logo flash for a split second, then disappear looks strange -- almost broken.  The render delay was added specifically to solve for this problem. Two seconds comes and goes in the blink of an eye, and it's just enough time for the preloader to justify it's existence as both a bridge and a branding mechanism.

If we can gaurantee the application will always loading in less than 2 seconds, than I'd sooner pull the preloader out entirely.  If we can't, then we need a preloader, and we should define a minimum display.

Reply:
It looks like something else is wrong on master, the first request for the spinner.png receives a 404 because its not properly included, the second insertion of the spinner happens after everything is loaded and would be ready to render. that being the reason for the spinner to be displayed too short, the supposed fix would make things even worse.

instead of increasing the delay, we need fo fix it on master, then the spinner should be displayed for ~2secs anyway, because it would be displayed while stuff is loading - not after the loading process is done. 👍

Reply:
@ashenden I understand your argument and agree that having the logo flash does indeed look strange but would argue that a "preloader" shouldn't be used as a bridge or branding mechanism. A "preloader" is only meant to indicate that something (loading) is happening as we otherwise risk that people assume that nothing is happening. 

I'd only display a spinner and optimize the UI to reduce the load time to a few milliseconds. What do you think?

Reply:
If showing a logo for a split second feels broken I'd opt in to remove the logo and show only the spinner. How'd you feel finding out that apple engineers have a 2 sec delay for no other reason but showing the logo in your iphone? :)

Reply:
Guys, let's not be dramatic here. You're all acting like a 2second delay is going to cause our customers to leave DC/OS.  This isn't a site that needs near instantaneous load time for it maintain customer retention and value.

FWIW, I'd be willing to bet that Apple bakes in a delay to it's iPhone and mac loading screens to ensure you see their logo for a couple seconds. 

I have a proposed compromise.  Let's lose the Mesosphere logo mark, and go with just the spinner -- limiting the delay to `1000ms`.  I do need one of you to look into why the image is error'ing and why it is taking so long to render in the DOM.

Reply:
I have fixed the 404 (using base64 inline code) and now the spinner is displayed for around 2.5 seconds (thats how long it takes to load our UI locally 🙀) - IMHO in this case your delay is okay (2s _overall_ loading time is okay, but _increasing_ our already long time with 2 _addidtional_ seconds was too much - and it tried to fix the wrong thing)

Reply:
@ashenden I agree that we probably don't need near-instantaneous load time to maintain customer retention and value, but we should, by all means, consider performance both during development as well as design. 

> Page load time and how fast your site feels is a large part of this user experience and should be weighed equally with the aesthetics of your site. 

/source  Designing for Performance by Lara Callender Hogan

My quick search for "Apple Logo Delay" seconds that, as it resulted in a long list people wondering whether their device is broken only because it took a bit longer to "load" than usual.

Showing only the spinner and limiting the delay to one second sound like a good compromise. Do you have the bandwidth to adjust this PR accordingly? Otherwise I'm more than happy to take care.

Reply:
if you could handle it, it would get done more quickly.

Reply:
Alright, I'll look into it.